gallery/src/components/NxTag/NxTagExample.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTag } from '@sonatype/react-shared-components';
+
+function NxTagExample() {
+  return (
+    <div className="test-nx-tag">
```

Comment:
What's this class for?

Reply:
> What's this class for?

Thought I needed it for a test. I didn't removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/816141f59b568c0155da5c781552a94a5ab278b0